src/Surfnet/StepupGateway/GatewayBundle/Controller/SecondFactorController.php

```diff
@@ -36,8 +36,7 @@
 use Symfony\Component\HttpKernel\Exception\BadRequestHttpException;
 
 /**
- * @SuppressWarnings(PHPMD.CouplingBetweenObjects) -- Too many second factor types in one controller. See Pivotal:
- *     https://www.pivotaltracker.com/story/show/104104610
+ * @SuppressWarnings(PHPMD.CouplingBetweenObjects)
```

Comment:
actually in my previous PR I fixed the TD issue but @DRvanR remarked that this issue was no longer relevant as it's better to group all second factors together in a controller so I removed the issue and the comment pointing there.